docs/products/clickhouse/reference/limitations.md

```diff
@@ -143,6 +143,19 @@ a
         or the [Aiven Console](https://console.aiven.io/).
       </td>
     </tr>
+    <tr>
+      <td>Maximum number of databases per service</td>
+      <td>
+          Your Aiven for ClickHouse service can support up to 400 databases simultaneously.
+      </td>
+      <td>
+        If your use case requires more than 400 databases, it's recommended to consolidate
+        data by adding a `client` column to your tables and including the column in the
+        primary key. This approach allows you to store all client data together while
+        maintaining separation. You can also limit scope of data on which SQL queries can
+        be run by using the `additional_table_filters` query setting.
+      </td>
```

Comment:
I suppose it might be misunderstood. Possible change:
```
If you are considering creating many databases of the same structure for isolation purposes, it's 
recommended to add extra column and filter by it instead. It will allow better scalability, and improve
robustness. A good idea would be to include this column into primary key or even partition by it.
You can also limit scope of data on which SQL queries can be run by using the `additional_table_filters`
query setting.
```